examples/kubernetes/templates/v1/cilium-crio-ds.yaml.sed

```diff
@@ -181,6 +181,33 @@ spec:
               add:
                 - "NET_ADMIN"
             privileged: true
+        - image: docker.io/cilium/cilium:latest
+          imagePullPolicy: Always
+          name: cilium-health
+          command: [ "cilium-health" ]
+          args:
+            - "-d"
+          livenessProbe:
+            exec:
+              command:
+              - cilium-health
+              - status
```

Comment:
You might want to consider using --succinct here too, the intention behind succinct is to print one line and succeed/fail (though I'm not sure it is quite acting as intended just yet.. but that's a matter for later).

Reply:
Done, added.